assignments/callbacks.js

```diff
@@ -21,26 +21,38 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length-1]);
 }
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y);
 }
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y);
 }
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  cb(item, function(items){
+    for (let i=0; i<list.length; i++){
```

Comment:
I appreciate it when you are using loops instead of inbuilt functions, it shows that you are getting comfortable in applying the logic.

Just to show other way you can use `includes` array function, remember that underneath it has the same logic you wrote.
```
function contains(item, list, cb) {
  if (!list.includes(item))
    return cb(false)
   return true
}